doc/man/usbguard.1.adoc

```diff
@@ -39,6 +39,8 @@ usbguard add-user 'name'
 
 usbguard remove-user 'name'
 
+usbguard check-IPC-permission
```

Comment:
Why are you making this a part of usbguard CLI? I thought this is just a way for developers to check IPC permissions before trying and possibly failing a call to daemon. In my opinion this should not be a part of CLI.